.travis.yml

```diff
@@ -34,8 +34,8 @@ before_install:
 
 install:
   - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
-      pip install codespell;
-      npm install -g bower;
+      pip install codespell awscli;
+      npm install -g bower asar;
```

Comment:
@lurch `asar` is required to create the package, while previously we were only running the test suite.